.travis.yml

```diff
@@ -1,4 +1,12 @@
 language: csharp
-sudo: false
+sudo: required
```

Comment:
This seems to prevent travis from [using containers](http://docs.travis-ci.com/user/workers/container-based-infrastructure/) which might slow down our builds.

Is there any reason that libuv needs to be installed globally? Couldn't we just install it anywhere in the path such as the working directory?